### ISSUE TITLE
Master WaterTAP3 issues

### ISSUE THREAD

**Issue by TimBartholomew** [watertap-org/watertap#447]
**Issues that were addressed**
- [x] #379 
- [x] Check for unused YAML files and clean up as necessary

**A list of issues that we want to address**
High priority:
- [x] Verify costing results for all unit operations (#664).
- [x] Ion exchange should have TDS removal updated - drop it to 0% for the default @kurbansitterley #795
- [x] Resolve valid ranges for costing UV - by either imposing bounds or reformulate the cost relationships (#426 ), add the UV costing from the detailed UV model
- [x] Review and clean up unused YAML files

Normal priority:
- [x] Handling of multiple units in parallel for zero order models (one unit that actually represents 3 in parallel - implications for costing) #572 
- [x] #347
- [ ] Documentation (Issue #595 )
- [ ] Ozone and Ozone AOP costing has high uncertainty @luohezhiming 
- [ ] ZO IX costing has high uncertainty - more detailed examples in WaterTAP could replace it @kurbansitterley 
- [ ] ZO GAC costing has high uncertainty - more detailed examples in WaterTAP could replace it @hunterbarber #882

Low priority:

- [ ] Assess whether chlorination should have a subprocess type in chemical addition
- [ ] There are many types of pump models, we may want to consider reorganizing them with more descriptive names and subprocess types - aggregate units representative of pumps that use pump_electricity function into some base unit like `pump_station` with corresponding yaml that comprises subtypes for each. Example units to consider: `municipal_drinking`, `sw_onshore_intake`, `well_field`, `deep_well_injection`, `backwash_solids_handling`; add `water_pumping_station` subtypes ("raw" and "treated")

Unprioritized:

**Unlikely to address**
- [ ] Only one of two methods for calculating evaporation in ponds is implemented

**Costing that has high uncertainty for zero order models**

**Comment by adam-a-a** [comment]
I think we should discuss chemical_addition units. External collaborators added one of these units to a flowsheet expecting the chemical to actually be added to the stream in their flowsheet.

**Comment by TimBartholomew** [comment]
Being broken out into separate issues
